### DISCUSSION TITLE
Handshake Failure

### DISCUSSION THREAD

**Discussion by patsch9** [DefGuard/defguard#675] [Q&A] [answered]
Hi, 
with defguard i allways become the message "Handshake did not complete" from wireguard client and cannot establish a connection. If i use the wireguard component with wg-quick or something the connection works. All Ports are open and the ip can be reached. I use it wirth ubuntu 22.04. 

Profile was added by scanning qr code.

Anyone has an idea?

**Comment by teon** [comment, marked as answer]
Hi, please follow these instructions: https://defguard.gitbook.io/defguard/contact-us/support
